src/main/resources/messages_id_ID.properties

```diff
@@ -1158,67 +1158,65 @@ split-by-sections.vertical.label=Pembagian Vertikal
-survey.button=Take Survey
-survey.dontShowAgain=Don't show again
+survey.nav=Survei
+survey.title=Survei Stirling-PDF
+survey.description=Stirling-PDF tidak memiliki pelacakan, jadi kami ingin mendengar dari pengguna kami untuk meningkatkan Stirling-PDF!
+survey.changes=Stirling-PDF telah berubah sejak survei terakhir! Untuk mengetahui lebih lanjut, silakan periksa posting blog kami di sini:
+survey.changes2=Dengan perubahan ini, kami mendapatkan dukungan bisnis yang dibayar dan pendanaan
+survey.please=Silakan pertimbangkan untuk mengikuti survei kami!
+survey.disabled=(Popup survei akan dinonaktifkan dalam pembaruan berikutnya tetapi tersedia di bagian bawah halaman)
+survey.button=Ikuti Survei
+survey.dontShowAgain=Jangan tampilkan lagi
 
 
 #error
-error.sorry=Sorry for the issue!
-error.needHelp=Need help / Found an issue?
-error.contactTip=If you're still having trouble, don't hesitate to reach out to us for help. You can submit a ticket on our GitHub page or contact us through Discord:
-error.404.head=404 - Page Not Found | Oops, we tripped in the code!
-error.404.1=We can't seem to find the page you're looking for.
-error.404.2=Something went wrong
-error.github=Submit a ticket on GitHub
-error.showStack=Show Stack Trace
-error.copyStack=Copy Stack Trace
-error.githubSubmit=GitHub - Submit a ticket
-error.discordSubmit=Discord - Submit Support post
+error.sorry=Maaf atas masalah ini!
+error.needHelp=Butuh bantuan / Menemukan masalah?
+error.contactTip=Jika Anda masih mengalami kesulitan, jangan ragu untuk menghubungi kami untuk bantuan. Anda dapat mengirim tiket di halaman GitHub kami atau menghubungi kami melalui Discord:
+error.404.head=404 - Halaman Tidak Ditemukan | Ups, kami tersandung dalam kode!
+error.404.1=Kami tidak dapat menemukan halaman yang Anda cari.
+error.404.2=Terjadi kesalahan
+error.github=Kirim tiket di GitHub
+error.showStack=Tampilkan Stack Trace
+error.copyStack=Salin Stack Trace
+error.githubSubmit=GitHub - Kirim tiket
+error.discordSubmit=Discord - Kirim pos dukungan
 
 
 #remove-image
-removeImage.title=Remove image
-removeImage.header=Remove image
-removeImage.removeImage=Remove image
-removeImage.submit=Remove image
-
-
-splitByChapters.title=Split PDF by Chapters
-splitByChapters.header=Split PDF by Chapters
-splitByChapters.bookmarkLevel=Bookmark Level
-splitByChapters.includeMetadata=Include Metadata
-splitByChapters.allowDuplicates=Allow Duplicates
-splitByChapters.desc.1=This tool splits a PDF file into multiple PDFs based on its chapter structure.
-splitByChapters.desc.2=Bookmark Level: Choose the level of bookmarks to use for splitting (0 for top-level, 1 for second-level, etc.).
-splitByChapters.desc.3=Include Metadata: If checked, the original PDF's metadata will be included in each split PDF.
-splitByChapters.desc.4=Allow Duplicates: If checked, allows multiple bookmarks on the same page to create separate PDFs.
-splitByChapters.submit=Split PDF
-
```

Comment:
readd these two empty lines, it needs all spacing to be same

Reply:
let me recheck, thanks

Reply:
done